crowbar_framework/app/models/backup.rb

```diff
@@ -75,14 +75,19 @@ def restore(options = {})
     from_upgrade = options.fetch(:from_upgrade, false)
 
     if Crowbar::Backup::Restore.restore_steps_path.exist?
+      logger.debug "Restore process is already running"
       errors.add(:base, I18n.t("backups.index.multiple_restore"))
       return false
     elsif !from_upgrade && backup_version != system_version
+      logger.debug "Restoring from different Crowbar version is not allowed"
       errors.add(:base, I18n.t("backups.index.version_conflict"))
       return false
     end
 
     if upgrade? && !upgrade
+      logger.debug(
+        "Upgrade from #{version} to #{ENV["CROWBAR_VERSION"]} is not supported"
+      )
```

Comment:
I guess it would be good to split the upgrade method up, to differentiate the error cases, but I'd prefer to not do it in this PR, would it be good enough for you if I create a card? right now in the worst case we would have a misleading log message